test_runner/src/main/kotlin/ftl/cli/firebase/test/android/AndroidRunCommand.kt

```diff
@@ -59,6 +60,9 @@ class AndroidRunCommand : CommonRunCommand(), Runnable {
     }
 
     override fun run() {
+
```

Comment:
hmmm do you have git pre commit working? :)

Reply:
Yes, but pre commit hook not catch that. I removed this line! Thanks!